packages/app-staking/src/Account/Bond.tsx

```diff
@@ -3,16 +3,23 @@
 // of the Apache-2.0 license. See the LICENSE file for details.
 
 import { I18nProps } from '@polkadot/ui-app/types';
+import { ApiProps } from '@polkadot/ui-api/types';
+import { CalculateBalanceProps } from '../types';
 
 import BN from 'bn.js';
 import React from 'react';
 import { AccountId } from '@polkadot/types';
+import { IExtrinsic } from '@polkadot/types/types';
 import { Button, InputAddress, InputBalance, Modal, TxButton, Dropdown } from '@polkadot/ui-app';
+import { withCalls, withApi, withMulti } from '@polkadot/ui-api';
+import { compactToU8a } from '@polkadot/util';
+import { SIGNATURE_SIZE } from '@polkadot/ui-signer/Checks';
```

Comment:
This is dangerous since it assumes that we are always signing with the publicKey - rather expose a `calcSignatureLength` that returns the value. (That we we don't need to find it all-over when we extend and cater for smaller transactions)